src/index.js

```diff
@@ -1 +1,21 @@
 import './style.css';
+import { displayScores, recordScore } from './game';
+
+displayScores();
+
+const submitScore = () => {
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
+
+  const data = {
+    user: name.value,
+    score: score.value,
+  };
+  recordScore(JSON.stringify(data));
+};
+
+const submitButton = document.getElementById('submitButton');
+submitButton.addEventListener('click', submitScore);
+
+const refreshButton = document.getElementById('refreshButton');
+refreshButton.addEventListener('click', displayScores);
```

Comment:
[OPTIONAL]

- After submitting the name and score, it's better to empty the input field again to make the UI better, and it will also help users that some event happened.

Reply:
Thank you Dipe, I will definitely implement this, it was just an oversight from my side.